src/components/tooltip/bl-tooltip.stories.mdx

```diff
@@ -3,17 +3,27 @@ import { ifDefined } from 'lit/directives/if-defined.js';
 import { styleMap } from 'lit/directives/style-map.js';
 import { unsafeHTML } from 'lit/directives/unsafe-html.js';
 import { Meta, Canvas, ArgsTable, Story, Preview, Source } from '@storybook/addon-docs';
+import { userEvent } from '@storybook/testing-library';
 
 <Meta
   title="Components/Tooltip"
   component="bl-tooltip"
+  parameters={{
+    layout: 'centered',
+    chromatic: { viewports: [800] },
+  }}
   argTypes={{
     placement: {
       control: 'text'
     }
   }}
 />
 
+export const tooltipOpener = async ({ canvasElement }) => {
+  const button = canvasElement.querySelector('bl-button')
+  await userEvent.click(button);
```

Comment:
I think click automatically triggers mouseenter event. We are having difficulties to simulate mouse events in integration tests. But click is also a valid trigger for tooltip since there is no mouse in touch devices. And luckily it works very well 😊